crate/operator/config.py

```diff
@@ -97,6 +97,12 @@ class Config:
     #: failed.
     CLUSTER_UPDATE_TIMEOUT = 7200
 
+    #: Time in seconds for which the operator will continue and wait to perform
+    #: a check if volume expansion has finshed successfully. Once this threshold
+    # has passed, a volume expansion is considered failed or not supported by the
+    # StorageClass.
+    EXPAND_VOLUME_TIMEOUT = 600
```

Comment:
It worked on both AWS/Azure within a couple of minutes, but I extended the timeout now to `1800` because the risk of blocking the whole process for too long seems to be quite low. Volume expansion seems to be supported quite well.